formats/text_messages_and_discord.html

```diff
@@ -0,0 +1,224 @@
+
+    ---
+
+    **Name of Poster 1**: [*Description of images, if any.*]
+
+    **Name of Poster 1**: Text of post.
+
+    >**Name of Poster 2**: Text of post.
+</format-block>
+
+<example-transcription
+        img="../images/message_opposite_sides_example.png"
+        alt="Screenshot of text messages where Mr Plow asks 'What was your in time? And how much snow is there?'. Other person
+        says 'We had to go pick up a machine, got here around 345. There is maybe an inch'. Mr Plow thanks them, other person
+        responses 'Finished up at af'. Mr Plow thanks them again."
+        extraTitle="Text Messages"
+>
+    <p>
+        <strong>Mr Plow</strong>: What was your in time? And how much snow is there?
+    </p>
+    <p>
+        ><strong>Red</strong>: We had to go pick up a machine, got here around 345. There is maybe an inch
+    </p>
+    <p>
+        <strong>Mr Plow</strong>: OK Thanks
+    </p>
+    <p>
+        ><strong>Red</strong>: Finished up at af
+    </p>
+    <p>
+        <strong>Mr Plow</strong>: Thanks
+    </p>
+</example-transcription>
+
+<example-transcription
+        img="../images/message_double_example.png"
+        alt="Screenshot of text messages where the person sends two consecutive messages to Mr Duckel. The messages read 'Do
+        you want coffee-milk or lemonade from the store?', 'puud?'."
+        extraTitle="Text Messages"
+>
+    <p>
+        ><strong>Green</strong>: Do you want coffee-milk or lemonade from the store?
+    </p>
+    <p>
+        ><strong>Green</strong>: puud?
+    </p>
+</example-transcription>
+
+<example-transcription
+        img="../images/message_with_paragraph_example.jpg"
+        alt="Screenshot of text messages where the person sends the message 'Gig' and the receiver, Hudson Valley Forward answers
+        with a message and expresses gratitude for assistance in portraying the transformation of the Hudson Valley's economy,
+        informing that starting Sept. 4, recipients will receive messages with occasional inquiries or requests to take photos of
+        their workspaces, aiming to connect with individuals outside the conventional 9-to-5 framework, and hinting at a forthcoming
+        networking event."
+        extraTitle="Text Messages"
+>
+    <p>
+        ><strong>Green</strong>: Gig
+    </p>
+    <p>
+        <strong>Hudson Valley Forward</strong>: Hey, it's Nick from the Poughkeepsie Journal. thanks for helping us tell the story
+        of the Hudson Valley's changing economy.
+    <p>
+        Starting Sept. 4, you'll receive messages from this number. Some days we'll ask a question. Other days we'll ask you to do
+        something simple like snap a photo of your work space.
+    </p>
+    <p> 
+        We're excited to meet with people like you who don't necessarily fit into a traditional 9-to-5 mold. And we're planning a
+        networking opportunity, so stay tuned for
+    </p>
+</example-transcription>
+
+<format-block title="Discord">
```

Comment:
For this section, it should just be the markdown copy-pasteable format, since it will have a "copy" button right underneath it.

Reply:
It is in markdown -- it's that it should be fill-out-able. Right now, if I click the 'Copy Format' button under it, I get this:

```
*Image Transcription: Discord*

---

**/u/SammyzABanana | UTC+10.5**

just mock this conversation

**/u/seeroflights•UTC-4 ~ 2 ⭐**

>**/u/fatalgift • UTC-4 •13 🌟**: hi everyone

hi fatal

**/u/cloakknight[mod]UTC-8 ~ 32 🌟**

>**/u/SammyzABanana | UTC+10.5**: just mock this conversation

no

**/u/cloakknight[mod]UTC-8 ~ 32 🌟**

>*Original message was deleted*

plz stop

**/u/SammyzABanana | UTC+10.5**

[*Image of a cat's face in front of a glowing green light.*]

**/u/fatalgift • UTC-4 •13 🌟**

>**/u/SammyzABanana | UTC+10.5**: *Click to see attachment*

this cat is SO green

wow

***/u/knightttime (UTC-5) ~ 14 🌟*** *is typing...*
```

So if I was going to use the Discord format, I'd have to delete this full transcription before I start typing, which is different from the first one:

```
*Image Transcription: Text Messages*

---

**Name of Poster 1**: [*Description of images, if any.*]

**Name of Poster 1**: Text of post.

>**Name of Poster 2**: Text of post.
```

which shows me what to do and what it should look like.

Reply:
Ohhhhh I get it now! I thought I was supposed to use the example template for that just as it was in our Reddit wiki. Gotchu! Can you check the last commit and lmk?

Reply:
The templates are supposed to be laid out that way already 😅 sounds like you just ran into a template that hasn't been updated in a while